app/views/post.py

```diff
@@ -0,0 +1,14 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import viewsets, permissions
+from rest_framework.response import Response
+from rest_framework.decorators import action
+
+from app.models import Post
+from app.serializers import PostSerializer
+
+class PostViewSet(viewsets.ModelViewSet):
+    serializer_class = PostSerializer
+    permission_classes = (permissions.AllowAny,)
```

Comment:
Let's start adding permissions here. I presume the user must be at least logged in to post something.